Changes candles and flamethrower

stobarico opened the pull request:
Thanks for TGStation and Partheo for the sprites ( Altought i made a bunch of vending machines on my own, since it was from yogstation )
There will be more soon!
https://i.gyazo.com/c3e8ddbd32c3080cb1a7a69741dc6fce.mp4
![a002ca1af2c5ebb7cdfc4e05b3a9677b](https://user-images.githubusercontent.com/15656746/35626136-e8dfb7aa-067b-11e8-8876-b36f9d0bd702.png)

stobarico committed:
Merge remote-tracking branch 'origin/dev' into dev

chaoko99 commented:
Oh no, people are going to ree at the prespective of those vending machines despite the fact that it is legitimately easier to sprite for.

chaoko99 commented:
Also: Change the P on the flamethrower to an H.

Zero-Bits commented:
Aye, the flammemwerfer uses Hydrogen now.

hey-it-stev commented:
Not keen on the bag sprites, the shape and perspective look very odd.

Anton-Kr commented:
:-1: on vending machines and body bags. Vending machines have the wrong perspective, and the body bags look worse and less clear then our current exisiting once.

BlueNexus commented:
Nice sprites, but the wrong perspective. We use a perspective related to the Two-Point/Three-Point perspectives (example: https://i.imgur.com/Ki1yYah.png )  while these use 3/4 (Only used by first aid kits.)

BlueNexus commented:
This'll need to be squashed.

chaoko99 commented:
[bAY CONTINUES TO HATE CHANGE, CHANGE THAT MAKES DEVELOPMENT EASIER]

You guys infuriate me. It is so much easier to sprite when there is *less to sprite.*

tlc2013 commented:
Objectively speaking, those bags look worse than the ones we currently have, so I'd prefer they stayed out. (Which, to be fair, they seem to be.)

Beside that? Everything's fine. I like it. I give no fucks about the perspective. So... yeah, cool.

chaoko99 commented:
I am indifferent to the bags. Not fond of the cryobag for sure though.

CrimsonShrike commented:
You say we use 3/4, yet a lot of stuff is either not possible to be displayed that way or has simply been ignored. Interesting.

stobarico committed:
Changes vending machines, bodybags, candles and flamethrower

Thanks for TGStation and Partheo for the sprites ( Altought i made a bunch of vending machines on my own, since it was from yogstation )
There will be more soon!
Changes "P" to "H"


Reverts bodybags and cryobags

because no one liked them
Reverts vending machines

I'm keeping those sprites for later
*wink* *wink*
Mine overmap spacemove icon proc optimizations

Unga bunga me use CHECK_TICK
Typeless loops too as I heard they help

stobarico commented:
Squashed, vending machines and bags have been removed due to lack of consistancy to the 3/4  A E S T H E T I C

BlueNexus commented:
We don't use 3/4. I was pointing out that these sprites use 3/4, whereas bay as a whole uses a perspective related to two-point and three-point.
Example of 3/4: http://static.tvtropes.org/pmwiki/pub/images/kakarikovillage.gif
Example of ours: https://i.imgur.com/Ki1yYah.png

chaoko99 commented:
Which is still idiotic. And you're not reading it right; they were removed.

3/4ths is easier to sprite for. By a large margin. Our perspective issue is a decade long holdover that nobody needed nor really should be gunning for. At worse, it forces us to resprite a ton of stuff that looked like ass anyways. At best: a lot of servers are making the jump because *gasp* it looks nicer and makes more sense for a top-down game.

I started this game thinking it was a side-perspective game, not top-down. Think about that for a bit.

stobarico commented:
To be honest, i do prefer 3/4, it looks way better than the current sprites most servers have shifted away from, maybe one day i will make a PR changing everything to 3/4, just to see the discussion it will create, however, i do agree that changing the vending machines ONLY, will create inconsistance.
